### PR TITLE
Doc changes for importing certs in controller

### PR DESCRIPTION
**Pull request by 1akshitha**
Added the documentation on the Importing SSL certificates in controller.

Related git issue: https://github.com/wso2/product-apim-tooling/issues/417

This is to cover the doc impact of https://github.com/wso2/product-apim-tooling/issues/235

### CODE REVIEW

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md [hunk 1]
```diff
@@ -813,4 +813,20 @@ Run the following CTL command to change the default location of the export direc
         - Required :   
             `--export-directory`: Path to directory where APIs should be saved.   
             Default : `/home/.wso2apictl/exported`
+            
+            
+## Import SSL Certificate for Secure HTTP Communication with API Manager
+
+Different environments of API Manager can have different SSL certificates for secure HTTP communications. The default
+certificate of WSO2 API Manager is a self signed certificate and in production environments it is advised to use a
+different certificate than default.
```
Comment (Chamindu36) [inline en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:822]:
```suggestion
certificate of WSO2 API Manager is a self-signed certificate and in production environments, it is advised to use a
different certificate than the default.  
```

Comment (Chamindu36) [inline en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:822]:
```suggestion
certificate of WSO2 API Manager is a self-signed certificate and in production environments, it is advised to use a
different certificate than the default.   
```

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md [hunk 2]
```diff
@@ -813,4 +813,20 @@ Run the following CTL command to change the default location of the export direc
...
             Default : `/home/.wso2apictl/exported`
+            
+            
+## Import SSL Certificate for Secure HTTP Communication with API Manager
+
+Different environments of API Manager can have different SSL certificates for secure HTTP communications. The default
+certificate of WSO2 API Manager is a self signed certificate and in production environments it is advised to use a
+different certificate than default.   
+
+If the certificate is the default WSO2 certificate or a CA signed certificate of a CA trusted by the OS, these
```
Comment (Chamindu36) [inline en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:824]:
```suggestion
If the certificate is the default WSO2 certificate or a CA-signed certificate of a CA trusted by the OS, these
```

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md [hunk 3]
```diff
@@ -813,4 +813,20 @@ Run the following CTL command to change the default location of the export direc
...
+certificates will be imported by default to the controller. If the CA or the certificate is new, you can add the
+certificate to the ```/home/.wso2apictl/certs``` directory. The certificates added to this directory will be imported 
+whenever an action is performed with the controller. Any DER or PEM encoded certificate can be used with controller.
```
Comment (Chamindu36) [inline en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:827]:
```suggestion
whenever an action is performed with the controller. Any DER or PEM encoded certificate can be used with the controller.
```

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md [hunk 4]
```diff
@@ -813,4 +813,22 @@ Run the following CTL command to change the default location of the export direc
...
+            
+## Import SSL Certificate for Secure HTTP Communication with API Manager
+
+Different environments of API Manager can have different SSL certificates for secure HTTP communications. The default
+certificate of WSO2 API Manager is a self-signed certificate and in production environments, it is advised to use a
+different certificate than the default.   
+
+If the certificate is the default WSO2 certificate or a CA-signed certificate of a CA (Certificate Authority) trusted by
+the OS, these certificates will be imported by default to the controller. If the CA or the certificate is new or does
+not get imported by default, you can add the certificate to the ```/home/.wso2apictl/certs``` directory. The
```
Comment (npamudika) [inline en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:826]: Now 3.2 onwards, we are allowing them to customize the root location of the .wso2apictl directory, so shall we reflect that here too?

Comment (npamudika) [reply en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:826]: The users can locate the .wso2apictl directory to another folder by setting an env variable, and of course the default will be the HOME directory

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md [hunk 5]
```diff
@@ -813,4 +813,22 @@ Run the following CTL command to change the default location of the export direc
...
+certificates added to this directory will be imported whenever an action is performed with the controller. Any DER or 
+PEM encoded certificate with file extensions of ```*.pem```, ```*.crt``` or ```*.cer``` can be used with the
```
Comment (npamudika) [inline en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:828]:
```suggestion
PEM encoded certificate with the file extensions of ```*.pem```, ```*.crt``` or ```*.cer``` can be used with the
```

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md [hunk 6]
```diff
@@ -813,4 +813,22 @@ Run the following CTL command to change the default location of the export direc
...
+controller. 
+
+!!! Info
+    If you are using windows, CA certs will not be imported by default and has to be added to the ```/home/.wso2apictl
+    /certs``` directory.
```
Comment (npamudika) [inline en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:833]: Same here

**Pull request merged**